state-chain/runtime/src/runtime_apis.rs

```diff
@@ -121,10 +121,10 @@ decl_runtime_apis!(
 		fn cf_suspensions() -> Vec<(Offence, Vec<(u32, AccountId32)>)>;
 		fn cf_generate_gov_key_call_hash(call: Vec<u8>) -> GovCallHash;
 		fn cf_auction_state() -> AuctionState;
-		fn cf_pool_price(from: Asset, to: Asset) -> Option<PoolPrice>;
+		fn cf_pool_price(base: Asset, quote: Asset) -> Option<PoolPriceV1>;
 		fn cf_pool_simulate_swap(
```

Comment:
Here you need the "from" and "to", or you need to add a side (i.e. intro a v2 of the rpc)